### PR TITLE
musescore 4

### PR DESCRIPTION
**Pull request by RustoMCSpit**
musescore studio is a future rebrand so update later

### CODE REVIEW

File: data/apps.json [hunk 1]
```diff
@@ -195,15 +195,25 @@
...
+  },
+    {
+    "name": "Musescore 4",
+    "url": "https://musescore.org/en",
+    "description": "A notation and sheet music app",
+    "repository": {
+      "type": "GitHub",
+      "user": "ejarzo",
+      "repo": "musical-garden"
+    }
```
Comment (satelllte) [inline data/apps.json:207]: you've swapped `ejarzo/musical-garden` & `musescore/MuseScore` repo links by accident

File: data/apps.json [hunk 2]
```diff
@@ -195,15 +195,25 @@
       "user": "LMMS",
       "repo": "lmms"
     }
+  },
+    {
+    "name": "Musescore 4",
```
Comment (satelllte) [inline data/apps.json:200]: I'd prefer to keep the name only (without being attached to any specific version)

```suggestion
    "name": "Musescore",
```

**Comment by RustoMCSpit** [comment]
i merged your suggestion, is this done

File: data/apps.json (hunk 1 above)
Comment (RustoMCSpit) [reply data/apps.json:207]: i merged your suggestion though, im very confused and dont think i know how to do this

Comment (satelllte) [reply data/apps.json:207]: @RustoMCSpit look at the full difference, as you can see the repository links are wrong. You need to swap `musical-garden` and `MuseScore` links

**Review by satelllte — APPROVED** [review]

**Comment by RustoMCSpit** [comment]
can it be merged

**Comment by RustoMCSpit** [comment]
why wasnt this merged

**Pull request merged**